scripts/rhoam-manifest-generator.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Can you add some detail about this script to the readme in the scripts folder. Details like how to use the script, any variables it may use or what the expected out come maybe.
I also have a question about the name of the script being misleading having the manifest in the name.

Reply:
Thanks @Boomatang - I'll add a description. How else would you like to see this script to be called?

Reply:
On calling the script I know you intended people use the make target but someone could call the script directly. 

Also a second point I wan to added in here. When I ran the script using the make target there was no output to tell me what happened. I didn't know what file or where it was created or if the scripted finished successfully. You may want to add some log statements.